tests/vppinputdecode.cpp

```diff
@@ -51,6 +51,14 @@ bool VppInputDecode::config(NativeDisplay& nativeDisplay)
     return status == DECODE_SUCCESS;
 }
 
+void getTimeStamp(SharedPtr<VideoFrame>& frame, VideoDecodeBuffer inputBuffer){
```

Comment:
Better make this as member function and return the time stamp.
because 
```
It may use member variable in future such as frame count.
according to file name it only change time stamp, so it's good to make timestamp as return value
```